### PR TITLE
Move raw data toggle out of visualization in CEO/LayoutCell

### PR DESCRIPTION
**Pull request by AlirieGray**
Closes: https://github.com/influxdata/applications-team-issues/issues/54

_What was the problem?_
Placing the `TimeMachineVis` component in the `LayoutCell` caused the Flux graph to overflow the cell. The raw data slide toggle also shifted the layout of components in `TimeMachine`.

_What was the solution?_
Moving the raw data slide toggle to the header above `TimeMachine` in data explorer and CEO, and changing the slide toggle to an option in the pencil drop-down menu in the `LayoutCell`.

  - [x] Rebased/mergeable
  - [ ] Tests pass

### CODE REVIEW

File: ui/src/shared/components/LayoutCell.tsx [hunk 1]
```diff
@@ -18,6 +18,11 @@ import {NoteVisibility} from 'src/types/dashboards'
 import {TimeSeriesServerResponse} from 'src/types/series'
 import {CellType} from 'src/types/dashboards'
 
+enum VisType {
+  Graph,
+  Table,
```
Comment (alexpaxton) [inline ui/src/shared/components/LayoutCell.tsx]: purely an aesthetic debate, but I wonder if this should be called "Raw"?

File: ui/src/shared/components/LayoutCellMenu.tsx [hunk 2]
```diff
@@ -11,6 +11,11 @@ import {Cell} from 'src/types/dashboards'
 import {QueryConfig} from 'src/types/queries'
 import {ErrorHandling} from 'src/shared/decorators/errors'
 
+enum VisType {
```
Comment (alexpaxton) [inline ui/src/shared/components/LayoutCellMenu.tsx]: Since you have this in 2 places, maybe move to an outside file and import into both?

Comment (AlirieGray) [reply ui/src/shared/components/LayoutCellMenu.tsx]: will do!

File: ui/src/shared/components/LayoutCell.tsx (hunk 1 above)
Comment (AlirieGray) [reply ui/src/shared/components/LayoutCell.tsx]: I personally think Graph/Table is more clear as far as what the user is going to see

**Pull request merged**